### PR TITLE
Mdapi processor

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/tests/mdapi.py [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+        'python-aniso8601',
+        'python3',
+        'django-markdown2',
+        'python-pytools',
+        'tyrus-container-servlet',
+        'python3-devel',
+        'waf-python3',
+        'avahi-ui-sharp-devel(x86-64)',
+        'python2-XStatic-roboto-fontface',
+        'python-Traits(x86-64)',
```
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/tests/mdapi.py]: Can we add some code to strip these ``(x86-64)`` strings from the end?  They'll cause pattern matching problems in other projects.

**Comment by ralphbean** [comment]
@pypingou I added some on to this.  Hope that's okay :)

Any :+1:'s?

**Comment by pypingou** [comment]
:+1: for me :)

**Comment by pypingou** [comment]
Let's merge :)

**Pull request merged**